### DISCUSSION TITLE
Support for conventionalcommits.org?

### DISCUSSION THREAD

**Discussion by HemangLavana** [PaulHatch/semantic-version#54] [General]
Hi,

I just came across your library and it looks very promising. I was wondering if it has support for:

https://www.conventionalcommits.org/en/v1.0.0/

I see that it supports MAJOR and MINOR keywords in commit message, but it would be nice to have support for the above.

Thanks,
Hemang.

**Comment by PaulHatch** [comment]
Yes, the major and minor patterns can be overridden and support regular expressions:

```
# A string which, if present in a git commit, indicates that a change represents a
 # major (breaking) change, supports regular expressions wrapped with '/'
 major_pattern: "(MAJOR)"
 # Same as above except indicating a minor change, supports regular expressions wrapped with '/'
 minor_pattern: "(MINOR)"
```
